src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpThurloeDAO.scala

```diff
@@ -97,17 +79,13 @@ class HttpThurloeDAO ( implicit val system: ActorSystem, implicit val executionC
 object HttpThurloeDAO {
   // Not private to allow this to be called by tests.
   def calculateExpireTime(profile:Profile): Long = {
-    (profile.lastLinkTime, profile.linkExpireTime) match {
-      case (Some(lastLink), Some(expire)) if (lastLink < DateUtils.nowMinus24Hours && expire > DateUtils.nowPlus24Hours) =>
-        // The user has not logged in to FireCloud within 24 hours, AND the user's expiration is
-        // more than 24 hours in the future. Reset the user's expiration.
-        DateUtils.nowPlus24Hours
-      case (Some(lastLink), Some(expire)) =>
+    profile.linkExpireTime match {
+      case Some(expire) =>
         // User in good standing; return the expire time unchanged
         expire
       case _ =>
-        // Either last-link or expire is missing. Reset the user's expiration.
-        DateUtils.nowPlus24Hours
+        // Expiration date is missing. Expire their access, forcing them to re-link
+        DateUtils.nowMinus1Hour
```

Comment:
isn't this whole function now `profile.linkExpireTime.getOrElse(DateUtils.nowMinus1Hour)`? Might as well use 0 instead of DateUtils.nowMinus1Hour.